widget/embedded/src/store/slices/config.ts

```diff
@@ -9,35 +10,71 @@ export const DEFAULT_CONFIG: WidgetConfig = {
   customDestination: true,
 };
 
-interface IframeConfigs {
+type IframeConfigs = {
```

Comment:
As we talked offline, please revert this `interface -> type` changes. for avoiding spamming your pr, i didn't put same comment on other places. please make sure you will update them as well.